lib/admin/upload_maximo.dart

```diff
@@ -302,6 +307,318 @@ Future<Map<String, List<List<String>>>> uploadToMaximo(
   return uploadData;
 }
 
+///Uploads assets to maximo.
+///
+///Order of upload: Location > Asset > JobPlan > JobLabor > JobAsset
+///
+///@returns Map<String, dynamic> with keys 'location', 'asset', 'jobplan', 'joblabor', 'jpassetlink'
+///
+///Possible statuses for each key: 'success', 'fail', 'duplicate'.
+///In case of duplicate, the upload will continue
+Future<Map<String, dynamic>> uploadAssetToMaximo(
```

Comment:
on second thought i would like to see one function for generating the upload, another one for actually doing the upload